ts-eks/test/klotho.yaml

```diff
@@ -5,3 +5,9 @@ out_dir: compiled
 defaults:
     execution_unit:
         type: eks
+        pulumi_params:
+            nodeType: node
```

Comment:
im wondering if we can just set them individually so we can test the node type as both fargate and node.

Also with the node constraints we have logic for when they differ and we want to make sure we launch the correct node groups and assign node selectors to the pods, so if we can also set this back to what it was before, we will properly be testing that